src/main/java/org/folio/service/mapping/processor/rule/DataSource.java

```diff
@@ -0,0 +1,49 @@
+package org.folio.service.mapping.processor.rule;
+
+import io.vertx.core.json.JsonObject;
+
+public class DataSource {
```

Comment:
It would be good if all these classes are moved to a json , and let the POJOs be generated

Reply:
The benefit of using schemas for POJO generating is just to provide support for marshalling/unmarshalling from POJO to request/response. We don't need such a benefit here. In addition, all this staff about records generating will be moved to a shared library without RMB-related things.   So, I don't think it is a good idea to use POJO generating here :)

Reply:
I saw the PR in a separated branch, looks good